content/04.baselines.md

```diff
@@ -3,6 +3,7 @@
 Before diving into a fancy thousand-layer neural network, always implement at least a simple model to establish an adequate performance baseline. 
 For example, researchers can build multinomial logistic regression or random forest models using the same software framework that is being used for DL and evaluate its classification performance. 
 This approach will help researchers with assessing the complexity of the task at hand and debugging more complex DL architectures. 
+The utility of these methods is evidenced by the recent development of hybrid models which combine DL and simpler models to improve robustness, interprability and confidence estimation [@arxiv:1803.04765; @arxiv:1805.11783].
 Depending on the amount of available data and the type of tasks, DL models may not necessarily be the best performing one. 
 As an illustration, the simple baseline models by Rajkomar A et al. [@doi:10.1038/s41746-018-0029-1] had achieved performance comparable with that of DL in a number of clinical prediction tasks using electronic health records, which may be a surprise to many. 
```

Comment:
> "models by Rajkomar A et al."

this should be 

- "Rajkomar et al."

or

- "A. Rajkomar et al."

or 

"Alvin Rajkomar *et al.*"

> the simple baseline models by Rajkomar A et al. [@doi:10.1038/s41746-018-0029-1] had achieved performance comparable

I believe here "had achieved" should be changed to "achieved" (or at least "have achieved")